en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md

```diff
@@ -495,17 +496,37 @@ LaTeX is installed, a beautifully formatted PDF file can be created
 using the same command structure:
 
 ```
-$ pandoc -o main.pdf main.md
+$ pandoc main.md -o main.pdf
+```
+
+If your document is written in languages other than English, you will likely need to use
+the XeLaTeX engine instead of plain LaTeX for .pdf conversion:
+
+```
+pandoc main.md --pdf-engine=xelatex -o main.pdf
 ```
 
-With time, you will be able to fine tune the formatting of PDF documents
-by specifying a LaTeX style file (saved to the same directory), and
-running something like:
+Make sure your tex editor supports the UTF-8 encoding. When using XeLaTeX for conversion into
+.pdf, instead of the `fontfamily` attribute in YAML, specify the `mainfont` attribute, to
+produce something like the following:
 
 ```
-$ pandoc -H format.sty -o project.pdf --number-sections --toc project.tex
+    ---
+    title: Plain Text Workflow
+    author: Dennis Tenen, Grant Wythoff
+    date: January 20, 2014
+    mainfont: times
+    ---
 ```
 
+Note that YAML often replicates some, although not all, of the command line argument (flag)
+functionality.  In this way, the font could be passed to Pandoc in the form of `pandoc main.md
+--mainfont=times -o target.pdf`. One could say "document formatting parameters are passed to
```

Comment:
"In this way..." and the following sentence "One could say..." are a convoluted expression of this aspect of Pandoc's argument parsing. Please tighten up this language.

Also, do you know if the Pandoc docs have a section that lists what subset of the cli args can be specified via YAML and which can't? I did a cursory browse but it wasn't totally clear to me - if you happen to know of a good listing, that'd be nice to add (but then, we might just be stuck with what we've got!)

Reply:
Will clean up the wording. The YAML vs CLI is not always clear. I'll send the reader to `man pandoc` in a footnote.